Story/RainMeadow.StoryHooks.cs

```diff
@@ -57,7 +57,28 @@ private void StoryHooks()
             On.SLOracleSwarmer.BitByPlayer += SLOracleSwarmer_BitByPlayer;
             On.CoralBrain.CoralNeuronSystem.PlaceSwarmers += OnCoralNeuronSystem_PlaceSwarmers;
             On.SSOracleSwarmer.NewRoom += SSOracleSwarmer_NewRoom;
+            On.HUD.TextPrompt.Update += TextPrompt_Update;
+        }
+
+        private void TextPrompt_Update(On.HUD.TextPrompt.orig_Update orig, TextPrompt self)
```

Comment:
There has to be a better place to do this than in update. You don't need to be Overwriting the gameoverstring every update cycle

Reply:
Unfortunately, the TextPrompt Update overwrites the gameoverstring both in the ctor and every Update cycle with the func, `UpdateGameOverString(string)` due to listening for what input the user has (controller, kb) and updates the message for how to initiate game over accordingly. If custom game over for clients  is set in the ctor alone, the Update frame would override it.

Reply:
Does hooking into updateGameOverString get over written?

Reply:
Good thinking. Can set the client game over string differently. Will still need the restartNotAllowed here to block clients from going to game over screen.

Reply:
Updated. Blocks clients from intiating, less invasive hooking, and better blocking on text prompt so it only happens during death screen ID

Reply:
Will update this weekend to allow clients to remove the death text so they can properly quit the game if they don't feel like waiting for the host.